release/version.txt

```diff
@@ -1 +1 @@
-2.0.1
+2.1.0
```

Comment:
I assume this is enough of a change to warrant a new minor release. If you don't agree or if this is not the right time to determine this please change it back.

Reply:
I will revert this on this branch to contina the scope of the PR. But we will adjust the version number for sure once we are ready to release.